spec/controllers/hq/users_controller_spec.rb

```diff
@@ -31,4 +31,74 @@ def foobar_action
     expect(response).to render_template 'index'
   end
 
+  describe '#edit and #update' do
+    before :each do
+      @old_user = FactoryGirl.build_stubbed(:user)
+      expect(User).to receive(:find).and_return(@old_user)
+    end    
+
+    specify 'editing renders the edit view' do
+      get :edit, id: @old_user.id
+      expect(response).to render_template 'edit'
+    end
+
+    it 'handles invalid updates' do
+      allow(@old_user).to receive(:save).and_return(false)
+      post :update, id: @old_user.id, user: { email: 'some email address'}
+      expect(response).to render_template 'edit'
+      new_user = controller.instance_variable_get(:@user)
```

Comment:
If this is what you intend to do, you may be able to `expect(assigns(:user).email).to eq 'string'`.  I see why you want to do this but I'm on the fence as to whether we should.  I'm okay either way